ansible_collections/arista/avd/tests/unit/filters/test_password.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import pytest
+
+from ansible_collections.arista.avd.plugins.filter.password import FilterModule, bgp_decrypt, bgp_encrypt, decrypt, encrypt
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import AristaAvdError, AristaAvdMissingVariableError
+
+from ..conftest import does_not_raise
+
+##########
+# BGP
+##########
+# NOTE: for now not testing anything with use_base64 for BGP False as this should not bew widely used
```

Comment:
I still don't follow why we want to support two different schemes here? When would you choose one or the other?

Reply:
let's make everything with usebase64=True and remove this confusion

Reply:
Addressed in 0dbeecf - removed all option to not use base64